students/ZachCooper/session04/mailroom2.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+from textwrap import dedent  
+
+# Create donors
+donors ={ 'fred jones': ("Fred Jones", [100, 200, 300]),
+        'amy schumer': ("Amy Schumer", [2000, 4000, 1000]),
+        'bill gates': ("Bill Gates", [5000, 10000, 30000]),
+        'chris fermstad' : ("Chris Fermstad", [300, 1000, 750]),
+        'ryan simmers': ("Ryan Simmers", [100000, 50000, 100]),
+        }
+
+        
+# Returns a list of current donors
+def print_donors():
+    print("Donor List:\n")
+    for donor in donors:
+        print(donors[0])
+
+# See if donor is in the list. Use case sensitive compare
+def find_donor(name):
```

Comment:
looks like this  isn't done yet -- do a new PR when ready for review.

And feel free to roll part 3 in while you are at it.